File: src/main/java/plus/maa/backend/config/external/CopilotBackup.java
package plus.maa.backend.config.external;

@lombok.Data
public class CopilotBackup {
    // 是禁用备份功能
    private boolean disabled;
    // 本地备份地址
    private String dir;
    // 远程备份地址
    private String uri;

    // git 用户名
    private String username;
    // git 邮箱
    private String email;
}

File: src/main/kotlin/plus/maa/backend/config/external/CopilotBackup.kt
package plus.maa.backend.config.external


data class CopilotBackup(
        /**
         * 是否禁用备份功能
         */
        var disabled: Boolean = false,

        /**
         * 本地备份地址
         * */
        var dir: String = "/home/dove/copilotBak",

        /**
         * 远程备份地址
         */
        var uri: String = "git@github.com:dragove/maa-copilot-store.git",

        /**
         * git 用户名
         */
        var username: String = "dragove",

        /**
         * git 邮箱
         */
        var email: String = "dragove@qq.com",
)
